src/components/dex/Pair.vue

```diff
@@ -45,6 +43,18 @@ export default {
         marketName: this.$store.state.currentMarket.marketName,
       });
     },
+    formatEntry(value, entry, key) {
+      if (key === '24H change') {
+        return `${this.$services.formatting.formatNumber(value)}%`;
+      }
+      return value;
+    },
+    getRelativeChange(value, entry, key) {
+      if (key === '24H change') {
```

Comment:
Since this is used twice, lets throw it in a `const`.

Reply:
Honestly, I'd rather leave it here for now because once we get this thing wired up using live data, this will definitely change since this came from my made up data. I def agree though, about the use of constants.